ckan/bootstrap.sh

```diff
@@ -19,6 +19,8 @@ fi
 rm -rf /usr/lib/adx/submodules/ckan/ckan/pastertemplates/template/ckanext_+project_shortname+.egg-info
 cd /usr/lib/adx/ || exit 1
 pipenv sync --dev
+echo "installed python packages"
+pipenv run python -m pip freeze
```

Comment:
Do we really need this? Does it provide any extra info compared to Pipfile.lock?

Reply:
So this was incredibly useful for me at one point. However, it may have been when we were still installing jinja2 using pip to get around the dependency conflict. Now we are using our own fork of frictionless until we move to ckan 2.10... so perhaps not?